src/sentry/static/sentry/less/forms.less

```diff
@@ -154,7 +154,14 @@ label {
 }
 
 .has-error .form-control {
-  border-color: @red-dark;
+  border-color: #c9c0d1;
+  box-shadow: inset 0 2px 0 rgba(0, 0, 0, 0.04);
+}
+
+.has-error .form-control:focus {
+  border-color: #a598b2;
```

Comment:
Should this be replaced with one of our known colors?

Reply:
this is a border color of our input fields, I just paste it here...but if we decide to update the fields in order to use the colors defined in the theme, I would do it in another PR